test/Elastic.Apm.Tests/MetricsTests.cs

```diff
@@ -101,7 +101,7 @@ public void ProviderWithException()
 		{
 			var mockPayloadSender = new MockPayloadSender();
 			var testLogger = new TestLogger(LogLevel.Information);
-			using (var mc = new MetricsCollector(testLogger, mockPayloadSender, new ConfigStore(new MockConfigSnapshot(), testLogger)))
+			using (var mc = new MetricsCollector(testLogger, mockPayloadSender, new ConfigStore(new MockConfigSnapshot(disableMetrics: "*"), testLogger)))
```

Comment:
That just makes this test run faster. This test tests that `MetricsCollector` can deal with `IMetricsProvider` implementations that throw an exception during metrics collection and also makes sure that we properly log that. Prior to this change we initialized `MetricsCollector` with all our default `IMetricsProvider`, but the test itself doesn't rely on any of those `IMetricsProvider` implementations. A few lines below we manually add one that always throws an exception to simulate errors in metric collection.

Adding `disableMetrics: "*"` makes the test run faster, because it does not initialize `IMetricsProvider`s that are not relevant for the test.